### ISSUE TITLE
What batch size and learning rate would you recommend training for a 24gb GPU?

### ISSUE THREAD

**Issue by yakovsushenok** [wuhuikai/FastFCN#112]
Is there some sort of cheat sheet on hyperparameter configuration that you maybe have?

**Comment by yakovsushenok** [comment]
In the paper and also in the examples in the readme, there is a GT model that performs quite well, however I don't see any references to it.
